### PR TITLE
Resolves "Extend Mapping Json Scheme with additional information"

### PR DESCRIPTION
**Pull request by vogtfa**
## Type of this PR 

- [ ] Bug fix
- [x] Enhancement

## Description

* The json schema for the mapping was extended with annotation fields (title, description, default). 
* Various corrections were made to the rest of the json schema.
* Some small changes to the javadoc comment were made for completeness or to fix grammar mistakes.

## Issue(s) related to this PR

 * Resolves issue #100

 
## Affected parts of the online documentation

None

## Definition of Done

- [x] You have read CONTRIBUTING.md carefully.
- [x] You have signed the [Contributor License Agreement](http://www.eclipse.org/legal/CLA.php).
- [x] All commits are signed-off (`-s`) with your mail address of your Eclipse Account.
- [x] `origin/main` has been merged into your Fork.
- [x] New functionality is covered by unit tests or integration tests. Code coverage must not decrease.
- [x] There are no new SpotBugs warnings. 
- [x] Coding guidelines have been observed (see CONTRIBUTING.md).
- [x] All tests pass.

### CODE REVIEW

File: fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json [hunk 1]
```diff
@@ -1,98 +1,124 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://example.com/schemas/abstract",
```
Comment (kschrab) [inline fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json:3]: is this line required?

Comment (vogtfa) [reply fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json]: No, this was just required for another jsonschema2md script. It is removed now.

File: fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json [hunk 2]
```diff
@@ -102,113 +127,184 @@
...
         },
         "server": {
+            "title": "server",
+            "description": "Object defining a server. Servers are a form of units that have no geographical location. The network properties of a server can be defined in the network.json-configuration in the cell-module.",
             "type": "object",
             "properties": {
-                "name": { "type": "string" },
-                "group": { "type": "string" },
+                "name": { 
+                    "description": "The name of the TMC to be matched against this object. All properties which are not specified will then be replaced.",
```
Comment (realmaxneu) [inline fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json:210]: Please change the whole description to:
> Used to be matched with a prototype. If a prototype name matches this name, all properties not set in this object will be overwritten by those defined in the prototype.

File: fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json [hunk 3]
```diff
@@ -102,113 +127,184 @@
...
+        "trafficLights": {
+            "title": "trafficLights",
+            "description": "Object defining a prototype for a traffic light. Since it is a traffic light only applications can be defined. Traffic light prototypes can be distributed among all traffic lights of an application by weight or assigned to specific traffic lights by using the ID of traffic light groups as reference.",
             "type": "object",
             "properties": {
-                "name": { "type": "string" },
-                "tlGroupId": { "type": "string" },
-                "group": { "type": "string" },
+                "name": { 
+                    "description": "The name of the prototype to be matched against this object. All properties which are not specified will then be replaced.",
```
Comment (realmaxneu) [inline fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json:230]: Please change the whole description to:
> Used to be matched with a prototype. If a prototype name matches this name, all properties not set in this object will be overwritten by those defined in the prototype.

File: fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json [hunk 4]
```diff
@@ -102,113 +127,184 @@
...
+                "group": { 
+                    "description": "The group name is used for (statistical) evaluation purposes with the StatisticOutput and ITEF. It allows to summarize multiple charginStation entities.",
+                    "type": "string" 
+                },
+                "access": { 
+                    "description": "Access restrictions, e.g. open to all or restricted to some communities, free of access or paying access.",
+                    "type": "string" 
+                },
+                "name": { 
+                    "description": "The name of the prototype to be matched against this object (will replace missing properties).",
```
Comment (realmaxneu) [inline fed/mosaic-mapping/src/main/resources/CMappingAmbassadorScheme.json:276]: Please change the whole description to:
> Used to be matched with a prototype. If a prototype name matches this name, all properties not set in this object will be overwritten by those defined in the prototype.

**Comment by realmaxneu** [comment]
Possibly more comments will follow...

**Pull request merged**